### DISCUSSION TITLE
root password needs to be entered multiple times during playbooks/setup-connections-complete.yml

### DISCUSSION THREAD

**Discussion by MagnaFrisia90** [HCL-TECH-SOFTWARE/connections-automation#342] [Q&A] [answered]
Hello,
during the ansible run I needed to enter the root password/password for the ssh-key multiple times.
If I did not react within a minute a timeout appeared and I needed re-run the script. Is this intended ?
The password is required during "was-dmgr-restart"-task and also during nodeagent-restart-task.

Can I somehow make the script/ansible playbook completely silent ? 
At the moment it is not even possible to shortly leave my desk to get some coffee without running into a timeout after starting the script.

![grafik](https://github.com/user-attachments/assets/4cf83a63-388d-4107-825c-b3408d7afc31)
![grafik](https://github.com/user-attachments/assets/c8104e7e-81b8-462a-8f85-93816e73fbb3)

Kind regards,
Jan

**Comment by stoeps13** [comment, marked as answer]
https://googlethatforyou.com?q=ansible%20ask%20for%20sudo%20password

**Reply by MagnaFrisia90** [reply]
als hätte ich das nicht schon längst getan...dann formuliere ich mal um:
How do you achieve avoiding re-entering the password for the ssh-key ? It seems like there are many ways.

**Reply by stoeps13** [reply]
Add the key to ssh-agent

https://stackoverflow.com/questions/50277495/how-to-run-an-ansible-playbook-with-a-passphrase-protected-ssh-private-key#51500802
